tests/serialize_test.py

```diff
@@ -38,6 +38,13 @@ def check_serde(self, s):
             b = v.as_bin()
             v1 = sexp_from_stream(io.BytesIO(b), to_sexp_f)
         self.assertEqual(v, v1)
+        # this copies the bytes that represent a single s-expression, just to
+        # know where the message ends. It doesn't build a python representaion
+        # of it
+        print(b)
```

Comment:
Probably want to remove the `print` at some point.

Reply:
oops. left-over debugging..